src/public/src/scripts/react/navbar/navbar.tsx

```diff
@@ -44,16 +48,16 @@ class Navbar extends React.Component<INavbarProps, void> {
      * @return {JSX.Element}
      */
     public render() : JSX.Element {
-            if (this.props.userPermission == PermissionLevel.SUPERADMIN) {
+            if (this.state.userPermission == PermissionLevel.SUPERADMIN) {
                 return(
                     <NavbarSuperAdmin />
                 );
-            } else if (this.props.userPermission == PermissionLevel.OWNER
-                || this.props.userPermission == PermissionLevel.ADMIN) {
+            } else if (this.state.userPermission == PermissionLevel.OWNER
```

Comment:
Non dovrebbe essere spostata sopra la condizione this.props.userPermission == PermissionLevel.OWNER? La navbar del SuperAdmin non è uguale a quella dell'Owner?